packages/react/src/Pagination/Pagination.tsx

```diff
@@ -196,53 +68,22 @@ function usePaginationPages({
       }
 
       return (
-        <Page {...props} key={key} theme={theme} className={clsx(enabled && classes.Page)}>
+        <span {...props} key={key} className={clsx(classes.Page)}>
```

Comment:
I thought this should have been an `a` tag based on the previous code, but that caused a test to fail with the expected number of `a` tags to render. Not sure if the test had to be updated, or if this is the correct default element.